### ISSUE TITLE
Logging logic is broken on macOS

### ISSUE THREAD

**Issue by raisedadead** [freeCodeCamp/tools-dashboard#7]
```
Retrieving PRs (#1 thru #1)
# of PRs Retrieved: 1
PR Range: 1 - 1
Data saved in file: data/openprs_1-1_2018-11-10T052600.json
Starting labeling process...
Successfully completed labeling
{ [Error: ENOENT: no such file or directory, open 'data/openprs_1-1_2018-11-10T052600.json']
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: 'data/openprs_1-1_2018-11-10T052600.json' }
{ [Error: ENOENT: no such file or directory, open 'data/open-prs-processed.json']
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: 'data/open-prs-processed.json' }
{ [Error: ENOENT: no such file or directory, open 'data/open-prs-processed.json']
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: 'data/open-prs-processed.json' }
{ [Error: ENOENT: no such file or directory, open 'data/open-prs-processed.json']
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: 'data/open-prs-processed.json' }
PR #1 successfully added a comment

PR #1 successfully added a comment

PR #1 successfully added a comment

PR #1 successfully added a comment
```

**Comment by RandellDawson** [comment]
I know what is causing the issue on your mac and will also affect the linux machine.  I need to correctly use path library to correctly working on all operating systems (Windows, Mac, Linux).  Maybe you can help me figure this logic out and you can submit a fix on a PR?

**Comment by raisedadead** [comment]
Closed in #8
